web/pdf_find_controller.js

```diff
@@ -500,7 +501,10 @@ class PDFFindController {
     if (this.#state === null || this.#shouldDirtyMatch(state)) {
       this._dirtyMatch = true;
     }
-    this.#state = state;
+    this.#state = {
+      ...state,
+      fuzzySearchEnabled: state.fuzzySearchEnabled ?? false,
+    };
```

Comment:
It worries me that we're making a change to a shallow copy here, as `#state` and `state` may need to share the same reference. It would take quite a bit of testing and analysis to verify that. Perhaps we just keep the same reference for sanity sake?

```suggestion
    this.#state = state;
    this.#state['fuzzySearchEnabled'] = state.fuzzySearchEnabled ?? false;
```

Reply:
Hmm, good point. I suppose I assumed that `this.#state` would always supersede `state` during code execution for these purposes, but yeah we can keep the same reference.

Reply:
Added that change manually